src/main/java/com/faforever/client/replay/ReplayService.java

```diff
@@ -143,6 +143,24 @@ static String parseMapName(byte[] rawReplayBytes) {
     return mapPath.split("/")[2];
   }
 
+  @VisibleForTesting
+  static String parseCoopMapName(byte[] rawReplayBytes) {
+    // "ScenarioFile" in hex
+    byte[] startPattern = new byte[] {0x53, 0x63, 0x65, 0x6E, 0x61, 0x72, 0x69, 0x6F, 0x46, 0x69, 0x6C, 0x65, 0x00, 0x01};
+    int mapStartIndex = Bytes.indexOf(rawReplayBytes, startPattern) + startPattern.length;
+    int mapEndIndex = 0;
+
+    for (int i = mapStartIndex; i < rawReplayBytes.length - 1; i++) {
+      //0x00 0x01 is the field delimiter
+      if (rawReplayBytes[i] == 0x00 && rawReplayBytes[i+1] == 0x01)
+        break;
```

Comment:
brackets

Reply:
Around the if ? You use brackets for one instruction ?

Reply:
Your convention be always use brackets